### PR TITLE
PIM-5604 : Add "remove_product_value" job definitions

### PR DESCRIPTION
**Pull request by momoss**
**Description**

What does this Pull Request do? reference the related issue?

**Definition Of Done**

| Q | A |
| --- | --- |
| Added Specs | N |
| Generate fixtures and install | Y |
| Generate products and import | Y |
| Review and 2 GTM | Y |
|  Tech Doc | N |

### CODE REVIEW

**Comment by SamirBoulil** [comment]
GTM

**Comment by willy-ahva** [comment]
GTM

**Comment by SamirBoulil** [comment]
RE-GTM

**Comment by willy-ahva** [comment]
GTM

File: Generator/JobGenerator.php [hunk 1]
```diff
@@ -115,13 +115,18 @@ protected function normalizeJobs(array $jobs)
...
-            "type"          => $job->getType(),
-            "configuration" => $job->getRawConfiguration()
+        $normalizeJob = [
+            "connector" => $job->getConnector(),
+            "alias"     => $job->getAlias(),
+            "label"     => $job->getLabel(),
+            "type"      => $job->getType()
         ];
+
+        if (count($job->getRawConfiguration()) > 0) {
```
Comment (phaseinducer) [inline Generator/JobGenerator.php:125]: `if (!empty($job->getRawConfiguration()))` ?

File: Resources/config/internal_jobs.yml [hunk 2]
```diff
@@ -44,7 +49,6 @@ jobs:
             delimiter:           ;
             enclosure:           '"'
             escape:              '\'
-            circularRefsChecked: true
```
Comment (phaseinducer) [inline Resources/config/internal_jobs.yml:47]: Why drop that?

File: Resources/examples/fixtures_enterprise.yml [hunk 3]
```diff
@@ -137,12 +137,12 @@ data_generator:
                     withHeader: true
                     filePath: /tmp/asset-category.csv
             approve_product_draft:
-                connector: Akeneo CSV Connector
+                connector: Akeneo Mass Edit Connector
```
Comment (phaseinducer) [inline Resources/examples/fixtures_enterprise.yml:140]: :+1:

**Comment by phaseinducer** [comment]
Little comments then GTM

File: Resources/config/internal_jobs.yml (hunk 2 above)
Comment (momoss) [inline Resources/config/internal_jobs.yml:47]: It creates "warning" when you install the PIM, and crash the install!

File: Generator/JobGenerator.php (hunk 1 above)
Comment (momoss) [inline Generator/JobGenerator.php:125]: Why not!

**Pull request merged**